action.yml

```diff
@@ -180,16 +180,21 @@ runs:
         export NVM_DIR="$HOME/.nvm"
         # this loads nvm into the current terminal
         [[ -s "$NVM_DIR/nvm.sh" ]] && \. "$NVM_DIR/nvm.sh"
-        nvm install
-        nvm use
-        rm -rf client/dist
-        npm install -g yarn
-        yarn install --network-concurrency 1
         if [[ -d vendor/silverstripe/admin ]]; then
           cd vendor/silverstripe/admin
+          nvm install
+          nvm use
+          npm install -g yarn
           yarn install --network-concurrency 1
           cd ../../..
         fi
+        nvm install
+        nvm use
+        rm -rf client/dist
+        if [[ $(npm -v) != $NPM_VERSION ]]; then
```

Comment:
fix this bit

Reply:
Have initialised the variable above.

Reply:
set `NPM_VERSION= ` above the `if [[ -d vendor/silverstripe/admin ]]; then` block in-case there is no admin dir

It ?might? work in bash as it is now, though it's weird and confusing

Reply:
And change the var name to `ADMIN_NPM_VERSION`

Reply:
Done